File: Hovercraft/Hovercraft.java
import java.util.Scanner;
public class Program
{
    public static void main(String[] args) {
Scanner in = new Scanner(System.in);
int sal = in.nextInt();
if(sal*3 == 21)
System.out.println("Broke Even");
else if(sal * 3 < 21)
System.out.println("Loss");
else
System.out.println("Profit");        
    }
}

File: Hovercraft/Hovercraft.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int sal = Convert.ToInt32(Console.ReadLine()) * 3;
if(sal == 21)
Console.Write("Broke Even");
else if (sal < 21)
Console.Write("Loss");
else
Console.Write("Profit");
        }
    }
}
